src/helpers/customCssStyleSheet.js

```diff
@@ -0,0 +1,31 @@
+import { forEach } from "./forEach";
+import { getCamelCase } from "./getCamelCase";
+export function CustomCssStyleSheet() {
```

Comment:
We should not have our properties in prototype. Let's move everything in a class